### PR TITLE
Simple Singolo

### PR DESCRIPTION
**Pull request by LeonidShatilo**
This branch adds files HTML, CSS, JS, and images for page Simple Singolo.

### CODE REVIEW

**Comment by Tanya92** [comment]
Оценка таски:

1. Вёрстка десктопной версии сайта 48/50:

- Our Services: иконки лучше делать иконочным шрифтом, 3 пункт про выход текста за границы не реализован

2. Адаптивная вёрстка с использованием медиазапросов: 100/100
3. Добавление интерактивности: 20/50
Итого: 168 баллов! 

Молодец, продолжай в том же духе!

File: assets/js/script.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  navigationButtons[index].addEventListener('click', (event) => {
+    navigationButtons.forEach((element) =>
+      element.classList.remove('active-button')
+    );
+    event.currentTarget.classList.add('active-button');
+  });
+}
+
+/* Scroll section */
+window.addEventListener('scroll', () => {
```
Comment (Tanya92) [inline assets/js/script.js:14]: These callbacks would be better to declare before adding listeners. Please read more about it

File: assets/js/script.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        [index].querySelector('a')
+        .classList.add('active-button');
+    }
+  });
+});
+
+/* Scroll animation */
+const anchors = document.querySelectorAll('a[href*="#"]');
+
+for (let anchor of anchors) {
```
Comment (Tanya92) [inline assets/js/script.js:37]: Why do you use for..of?

File: assets/js/script.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  });
+});
+
+/* Mixed portfolio images */
+const filterList = document.getElementById('filter');
+let arr1 = imagesPortfolio.querySelectorAll('img');
+let a = 0;
+let arr2 = [];
+for (var i = arr1.length - 1; i >= 0; i--) {
+  arr2[i] = a;
```
Comment (Tanya92) [inline assets/js/script.js:78]: Could you understand what this code does? (strings 74-79)

File: assets/js/script.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+}
+filterList.querySelectorAll('li').forEach((el) => {
```
Comment (Tanya92) [inline assets/js/script.js:80]: if you use only one variable for a function you don't need to using () . el => {...}

File: assets/js/script.js (hunk 1 above)
Comment (LeonidShatilo) [reply assets/js/script.js:14]: Thank you for the remark. I will take it into account.

File: assets/js/script.js (hunk 2 above)
Comment (LeonidShatilo) [reply assets/js/script.js:37]: In order to perform a concise iteration of elements. :)

File: assets/js/script.js (hunk 3 above)
Comment (LeonidShatilo) [reply assets/js/script.js:78]: By using the querySelectorAll method I find all elements with the img selector and assign arr1 to the variable. I assign an empty array to the variable arr2. Then, using the for loop, I fill the arr2 array with zeroes, the number of which is equal to the number of found img selectors.

File: assets/js/script.js (hunk 4 above)
Comment (LeonidShatilo) [reply assets/js/script.js:80]: Okay. I will take it into consideration.

**Comment by LeonidShatilo** [comment]
Thank you for checking and evaluating my assignment!

**Pull request merged**